apps/prairielearn/src/pages/instructorRequestCourse/instructorRequestCourse.html.ts

```diff
@@ -0,0 +1,247 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+import { type CourseRequestRow } from './instructorRequestCourse';
+
+export const RequestCourse = ({
+  course_requests,
+  resLocals,
+}: {
+  course_requests: CourseRequestRow[];
+  resLocals: Record<string, any>;
+}) => {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head')%>", {
+          ...resLocals,
+        })}
+        <script>
+          $(function () {
+            $('input[name=cr-role]').change(function () {
+              var role = this.value;
+              $('.question-form button').prop('disabled', role != 'instructor');
+              $('.role-comment').hide();
+              $('.role-comment-' + role).show();
+            });
+          });
```

Comment:
For real modernization we'd also remove all use of jQuery 😄. It's fine to leave it for now, but I also wouldn't object to getting rid of it. There are various [cheat sheets](https://tobiasahlin.com/blog/move-from-jquery-to-vanilla-javascript/) on the web for this.

Reply:
I'm going to hold off on that because I'm trying to not change the functionality. I feel like jQuery cleanup would want to be more consistent across multiple places where similar patterns are used and I'm trying to keep this PR scoped down.